boot/arch/abs32le/meson.build

```diff
@@ -0,0 +1,3 @@
+BUILD = false
```

Comment:
Please add copyright headers to all added files.

Reply:
Will do.

Reply:
I mostly omitted them so far because files made 90% out of copyright header make git's move detection mess up diffs.